source/usage-examples/bulkWrite.txt

```diff
@@ -0,0 +1,86 @@
+=======================
+Perform Bulk Operations
+=======================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+MongoDB provides clients the ability to perform write operations in bulk.
+Bulk operations affect a *single* collection.
+
+The following code sample performs a bulk write operation against the
```

Comment:
Our other usage examples have tried to flesh out the methodology before bringing up the code samples. For reference, we're using this document: https://docs.google.com/document/d/1s4C1gA0OiNWgLeYzIwTvBsT1D_XcP-7A-Lb5WNUmUT4/edit?folder=14KsXkgiGp4uu14nb0hBc45qwbMePMZCA#heading=h.fqr4s89j4ciy
to track how we're structuring these. It might be a good idea to move the "following code" example description down into the example section and keep the bulkWrite reference up here.